knowledge_repo/app/static/css/pages/base.css

```diff
@@ -31,6 +38,20 @@
         display: flex!important;
     }
 }
+
+/* Nav bar search form design */
+.form-control {
+  border-radius: 21px;
+  box-shadow: 0 0px 0px rgba(0,0,0,0);
```

Comment:
I think this can be `0 0 0 black` instead? But actually, if this is unsetting box-shadow, you might be able to set it to `unset` instead

Reply:
`unset` worked!